less/home-page.less

```diff
@@ -1,21 +1,58 @@
+
+
+
+
+
 .home {
+  padding: 0 40px;
 
+  .photoheader {
```

Comment:
The stlying here for `.photoheader` doesn't add much to the page. You could remove lines 9-13 and your code would be a little cleaner without harming the webpage's design.